src/ripple/app/ledger/Ledger.cpp

```diff
@@ -416,7 +416,7 @@ Ledger::setAccepted(NetClock::time_point closeTime,
 bool Ledger::addSLE (SLE const& sle)
 {
     SHAMapItem item (sle.getIndex(), sle.getSerializer());
-    return stateMap_->addItem(item, false, false);
+    return stateMap_->addItem(std::move(item), false, false);
```

Comment:
Ok.  I'm going to hold off on marking this Passed until b4 gets pushed and then I'll rebase this to that and retest.  If all goes well, then I'll Pass it.